src/components/ItaliaTheme/Blocks/Common/SearchFilters/TextFilter.jsx

```diff
@@ -8,26 +9,52 @@ const messages = defineMessages({
   },
 });
 
-const TextFilter = ({ value, id, onChange, placeholder, blockID }) => {
+const TextFilter = ({
+  value,
+  id,
+  onChange,
+  placeholder,
+  blockID,
+  activeTopLabel = false,
+  infoText = '',
+}) => {
   const intl = useIntl();
+  const placeholderText =
+    placeholder || intl.formatMessage(messages.text_filter_placeholder);
 
   return (
     <div className="me-lg-3 my-2 my-lg-1 filter-wrapper text-filter">
-      <label htmlFor={`${blockID}-${id}`} className="visually-hidden">
-        {placeholder}
-      </label>
-      <input
-        type="text"
-        placeholder={
-          placeholder || intl.formatMessage(messages.text_filter_placeholder)
-        }
-        value={value}
-        onChange={(e, data) => {
-          onChange(id, e.target.value ?? '');
-        }}
-        autocomplete="off"
-        id={`${blockID}-${id}`}
-      />
+      {!activeTopLabel ? (
+        <>
+          {/* Input con label NACOSTA */}
+          <label htmlFor={`${blockID}-${id}`} className="visually-hidden">
+            {placeholderText}
+          </label>
+          <input
+            type="text"
+            placeholder={placeholderText}
+            value={value}
+            onChange={(e, data) => {
+              onChange(id, e.target.value ?? '');
+            }}
+            id={`${blockID}-${id}`}
+          />
+          {infoText && <small className="form-text d-block">{infoText}</small>}
+        </>
```

Comment:
secondo me questo si può fare (se la versione di design-react-kit che stiamo usando lo permette) con un semplice <Input label={placheholderText} [....] />

secondo questa documentazione https://design-react-ml1g4w3a9-dip-trasformazione-digitale.vercel.app/?path=/docs/documentazione-form-input--documentazione

dovrebbe mettere il placeholder e quando si clicca sull input la label viene translata sopra

Reply:
@giuliaghisini forse tu intendevi come è stato fatto qui nel blocco di ricerca quando abbiamo customizzato la SearchInput? https://github.com/RedTurtle/design-comuni-plone-theme/blob/main/src/customizations/volto/components/manage/Blocks/Search/components/SearchInput.jsx
Onestamente però il codice non lo vedo molto pulito, tra l'altro l'effetto label sopra è molto scattoso perchè appunto da un placeholder passa a una label
https://v3.io-comune.redturtle.it/wagner-workroom-factory/cerca

Capiamo insieme come procedere

Reply:
quello che intendevo io era che forse non era necessario riscrivere tutto l'html della label e dell'input perchè probabilmente già design-react-kit lo fa.
Se hai provato con il componente di design-react-kit e hai visto che non funziona, allora va bene cosi